Makefile

```diff
@@ -1,5 +1,7 @@
 TEST_RUN_EXTRA_PARAMS?=
 PYTHON?=python
+MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+PROJECT_DIR := $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
```

Comment:
It is not used anywhere.

Reply:
removed both `MAKEFILE_PATH` and `PROJECT_DIR`

Reply:
revert back removing of `MAKEFILE_PATH`, it's needed to run tests with `coverage.py`

Reply:
the same for `PROJECT_DIR`, used to set `PYTHONPATH`

Reply:
Okay.